okhttp-android-support/src/test/java/okhttp3/internal/huc/ResponseCacheTest.java

```diff
@@ -269,6 +270,8 @@ private void testResponseCaching(TransferKind transferKind) throws IOException {
   }
 
   @Test public void secureResponseCaching() throws IOException {
+    assumeFalse(getPlatform().equals("jdk9"));
```

Comment:
It's a good idiomatic improvement. In this case getPlatform() can't be null because of it's implementation System.getProperty("okhttp.platform", "platform"), but your point is valid.

For now, I'd rather keep this inline with other similar usages, and change all in one go if we want to.

```
$ git grep getPlatform...equals | awk -F / '{print $NF}'
ResponseCacheTest.java:    assumeFalse(getPlatform().equals("jdk9"));
ResponseCacheTest.java:    assumeFalse(getPlatform().equals("jdk9"));
URLConnectionTest.java:    assumeTrue(getPlatform().equals("jdk9"));
Jdk9PlatformTest.java:    assumeTrue(getPlatform().equals("jdk9"));
Jdk9PlatformTest.java:    assumeTrue(getPlatform().equals("jdk9"));
JdkWithJettyBootPlatformTest.java:    assumeTrue(getPlatform().equals("jdk-with-jetty-boot"));
CertificatePinnerChainValidationTest.java:    if (getPlatform().equals("jdk9")) {
CertificatePinnerChainValidationTest.java:    if (getPlatform().equals("jdk9")) {
UrlConnectionCacheTest.java:    assumeFalse(getPlatform().equals("jdk9"));
UrlConnectionCacheTest.java:    assumeFalse(getPlatform().equals("jdk9"));
```